server/api/api.go

```diff
@@ -109,8 +109,10 @@ func (r Router) Handler() http.Handler {
 
 	cors := cors.New(corsOpts)
 	router.Use(cors.Handler)
-
 	router.Mount("/api/v1", r.apiRouter())
+
+	router.Get("/archive/{id}/", build.HandleArchive(r.Jobs, r.Config))
```

Comment:
I need extra help with this line - it doesn't work in a way in which I'd expect it to work... I get a 404 instead of the response from function